vmray/help.md

```diff
@@ -524,8 +525,9 @@ _This plugin does not contain any troubleshooting information._
 
 # Version History
 
+* 5.0.0 - New Titles for output data (spelling corrections)
```

Comment:
keep version at 4.0.2 for now since 4.0.2 hasn't been released and title corrections only warrant a patch version bump anyway